include/aos/common/crypto.hpp

```diff
@@ -83,17 +79,184 @@ constexpr auto cSerialNumSize = AOS_CONFIG_CRYPTO_SERIAL_NUM_SIZE;
 constexpr auto cSerialNumStrLen = AOS_CONFIG_CRYPTO_SERIAL_NUM_STR_LEN;
 
 /**
- * General certificate private key type.
+ * RSA modulus size.
+ */
+constexpr auto cRSAModulusSize = AOS_CONFIG_CRYPTO_RSA_MODULUS_SIZE;
+
+/**
+ * Size of RSA public exponent.
+ */
+constexpr auto cRSAPubExponentSize = AOS_CONFIG_CRYPTO_RSA_PUB_EXPONENT_SIZE;
+
+/**
+ * ECDSA params OID size.
+ */
+constexpr auto cECDSAParamsOIDSize = AOS_CONFIG_CRYPTO_ECDSA_PARAMS_OID_SIZE;
+
+/**
+ * DER-encoded X9.62 ECPoint.
+ */
+constexpr auto cECDSAPointDERSize = AOS_CONFIG_CRYPTO_ECDSA_POINT_DER_SIZE;
+
+/**
+ * Supported key types.
+ */
+enum class KeyType { eRSA, eECDSA };
+
+/**
+ * Public key interface.
+ */
+class PublicKey {
```

Comment:
Add Itf suffix for all interfaces. I missed it in the previous PR, so please add it for PrivateKey interface as well.